docs/core/tools/dotnet-clean.md

```diff
@@ -36,19 +40,19 @@ Prints out a short help for the command.
 
 `-o|--output <OUTPUT_DIRECTORY>`
 
-Directory in which the built binaries were placed. You also need to define `--framework` when you specify this option. If you didn't specify this property during build time, you don't have to specify it for clean.
+Directory in which the build outputs have been placed.
```

Comment:
It's on the framework switch, but I added a line back to cover it on the output switch.